src/Psr3Yii2Logger.php

```diff
@@ -21,41 +22,37 @@ public function __construct()
      * Log a message.
      *
      * @param mixed $level One of the \Psr\Log\LogLevel::* constants.
-     * @param string $message The message to log, possibly with {placeholder}s.
+     * @param string|array $message The message to log, possibly with {placeholder}s.
      * @param array $context An array of placeholder => value entries to insert
      *     into the message.
-     * @return null
+     * @return void
      */
     public function log($level, $message, array $context = [])
     {
-        $messageToLog = $this->interpolate($message, $context);
+        if (is_array($message)) {
+            $messageToLog = $this->interpolateArray($message, $context);
+        } else {
+            $messageToLog = $this->interpolate($message, $context);
+        }
         switch ($level) {
-            case PsrLogLevel::EMERGENCY:
-                Yii::error($messageToLog);
-                break;
             case PsrLogLevel::ALERT:
-                Yii::error($messageToLog);
-                break;
             case PsrLogLevel::CRITICAL:
-                Yii::error($messageToLog);
-                break;
+            case PsrLogLevel::EMERGENCY:
             case PsrLogLevel::ERROR:
                 Yii::error($messageToLog);
                 break;
             case PsrLogLevel::WARNING:
                 Yii::warning($messageToLog);
                 break;
             case PsrLogLevel::NOTICE:
-                Yii::info($messageToLog);
-                break;
             case PsrLogLevel::INFO:
                 Yii::info($messageToLog);
                 break;
             case PsrLogLevel::DEBUG:
-                Yii::trace($messageToLog);
+                Yii::debug($messageToLog);
```

Comment:
Why the change from `Yii::trace()` to `Yii::debug()`?

Reply:
trace is deprecated.